meta.json

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
landrbot will automatically generate this file, so it can be removed

Reply:
sweet, will it be added to source control too? I guess it's not necessary

Reply:
@tmigone Nope, it won't be under source control